langchain4j-core/src/main/java/dev/langchain4j/agent/tool/ReturnAsOutput.java

```diff
@@ -0,0 +1,19 @@
+package dev.langchain4j.agent.tool;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Java methods annotated with @Tool and @ReturnAsOutput are considered tools that language model can use,also Directly use tool funcion return value as the output of the model
+ * When using OpenAI models, <a href="https://platform.openai.com/docs/guides/function-calling">function calling</a>
+ * is used under the hood.
+ */
+
+@Target(METHOD)
+@Retention(RUNTIME)
+public @interface ReturnAsOutput {
```

Comment:
What about adding an attribute to existing `@Tool` annotation instead of introducing a new annotation?

`@Tool(returnDirectly = true)`

We need to think about a better name though

Reply:
this is a better choice

Reply:
> this is a better choice

Not sure I understand :) To what exactly "this" refers?

Reply:
> > this is a better choice
> 
> Not sure I understand :) To what exactly "this" refers?

my bad ,I mean adding an attribute to existing @Tool annotation instead of introducing a new annotation is a better choice ,I will try to optimize my code